src/lib/componentloader.js

```diff
@@ -22,8 +22,12 @@ ComponentLoader.prototype = {
     xhr.open('GET', registryBaseUrl + getMajorVersion(AFRAME.version) + '.json');
 
     xhr.onload = () => {
-      this.components = JSON.parse(xhr.responseText).components;
-      console.info('Components in registry:', Object.keys(this.components).length);
+      if (xhr.status === 404) {
+        console.error('Error loading registry file: 404');
+      } else {
+        this.components = JSON.parse(xhr.responseText).components;
+        console.info('Components in registry:', Object.keys(this.components).length);
```

Comment:
Do we want a better message here? The error message shows what happened but success does not.
"Registry file loaded. " + Object.keys(this.components).length + " components available"